templates/articles/Estimating an Agile Projects Scope.json

```diff
@@ -0,0 +1,7 @@
+{
+  "title": "Estimating an Agile Projects Scope",
+  "author": "Melanie Key",
+  "date": "2023-09-24",
+  "subpath": "Estimating an Agile Projects Scope",
```

Comment:
I think we'll want this to not include spaces, since it'll be in the URL for the blog post.